src/Adapter/Product/QueryHandler/GetProductForEditingHandler.php

```diff
@@ -223,7 +124,8 @@ public function handle(GetProductForEditing $query): ProductForEditing
             $this->getAttachments($query->getProductId()),
             $this->getProductStockInformation($product),
             $this->getVirtualProductFile($product),
-            $this->getCover($query->getProductId(), $product->getShopId())
+            $this->getCover($query->getProductId(), $product->getShopId()),
+            $this->getDateNew($product)
```

Comment:
If the field is in the Options tab, then this data should probably be in the Options sub object in query result as well to remain consistent with other fields